### DISCUSSION TITLE
Starred topics on Explore page

### DISCUSSION THREAD

**Discussion by burnsra** [community/community#21806] [New to GitHub]
<p>I believe the UI for starred topics on the explore page is broken. Links go to /topic.{topicname} instead of /topic/{topicname}, which results in 301’s to the general /topic page.</p>

**Comment by logankilpatrick** [comment]
<p>Hey <a class="mention" href="/burnsra">@burnsra</a>, mind giving an example of a query or link that returns a 301? I am able to view my starred topics and click through to the repos with those topics. </p>

**Comment by burnsra** [comment]
<p>Please see the macOS topic link below in the screenshot, it links to “/topic.macos”.  The request/response is as follows:</p>
<p>Request</p>
<p>Request URL: <a href="https://github.com/topic.macos" rel="nofollow noopener">https://github.com/topic.macos</a><br>
Request Method: GET<br>
Status Code: 301 Moved Permanently</p>
<p>Response</p>
<p>Location: <a href="https://github.com/topics" rel="nofollow noopener">https://github.com/topics</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e82c6f5336ab38dd0cd498cfa1d0ab2d27c70e54.png" data-download-href="/uploads/short-url/x7TYpAPRC5Qc1e40wqfaKyTUr88.png?dl=1" title="Screen Shot 2020-03-07 at 8.25.09 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/171054/181066965-525c8ce7-45e7-40fc-8e7f-568a9edb99b9.png" alt="Screen Shot 2020-03-07 at 8.25.09 AM.png" data-base62-sha1="x7TYpAPRC5Qc1e40wqfaKyTUr88" width="666" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e82c6f5336ab38dd0cd498cfa1d0ab2d27c70e54_2_666x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e82c6f5336ab38dd0cd498cfa1d0ab2d27c70e54_2_999x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/e82c6f5336ab38dd0cd498cfa1d0ab2d27c70e54_2_1332x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e82c6f5336ab38dd0cd498cfa1d0ab2d27c70e54_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-03-07 at 8.25.09 AM.png</span><span class="informations">2048×1536 527 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by logankilpatrick** [comment]
<p>Ah, yes, I experienced that as well when I was testing things before I replied to you. I suggest contacting support to report this as it’s not clear if this is by design or not. </p>

**Comment by burnsra** [comment, marked as answer]
<p>This appears to be fixed as of today, Monday, March 9th.</p>
